### PR TITLE
Updated axe core version of accessibility-insights-service to 4.8.4

### PR DESCRIPTION
**Pull request by SaanicaG**
#### Details
Update axe core version of accessibility insights service to 4.8.4

#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Added relevant unit test for your changes. (`yarn test`)
- [ ] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [ ] Ran precheckin (`yarn precheckin`)
- [ ] Validated in an Azure resource group

### CODE REVIEW

File: packages/scanner-global-library/src/axe-scanner/axe-puppeteer-scanner.ts [hunk 1]
```diff
@@ -47,7 +47,7 @@ export class AxePuppeteerScanner {
 
         if (
             page.navigationResponse.request()?.redirectChain()?.length > 0 ||
-            // Should compare encoded Urls
+            // Should compare encoded urls
```
Comment (lamaks) [inline packages/scanner-global-library/src/axe-scanner/axe-puppeteer-scanner.ts:50]: Abbreviation, should be `URLs`

**Comment by v-rakeshsh** [comment]
Hi @v-viyada can we close this PR?? as build is not triggering because of improper branching. Please suggest?

**Comment by v-viyada** [comment]
> Hi @v-viyada can we close this PR?? as build is not triggering because of improper branching. Please suggest?

please close this and link the correct PR to this.

**Comment by lamaks** [comment]
Please close this PR in favor #2531

**Comment by v-rakeshsh** [comment]
Hi all, closing this PR as new PR [2531](https://github.com/microsoft/accessibility-insights-service/pull/2531) is created for same.
Thank you.

**Comment by v-rakeshsh** [comment]
Hi all, closing this PR as new PR [2531](https://github.com/microsoft/accessibility-insights-service/pull/2531) is created for same.
Thank you.

**Pull request closed without merging**